### PR TITLE
Convert project from Yarn to npm

### PR DESCRIPTION
**Pull request by wrumsby**
Using [synp](https://github.com/imsnif/synp) converted project from Yarn to npm since npm is our standard tooling.

### CODE REVIEW

**Comment by lwcooper** [comment]
did you want an `.nvmrc` file too?

File: src/ServerlessOpenApiDocumentation.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import * as c from 'chalk';
+import chalk from 'chalk';
```
Comment (lwcooper) [inline src/ServerlessOpenApiDocumentation.ts:1]: looks like more than just updating from yarn to npm?

File: test/project/openapi.yml [hunk 2]
```diff
@@ -143,7 +143,7 @@ components:
 info:
   title: ''
   description: ''
-  version: 85bfbe6d-64b2-45e3-b74e-d26ba8dfa3bb
+  version: dc173422-424a-494c-a98e-98f16814d721
```
Comment (lwcooper) [inline test/project/openapi.yml:146]: is this auto generated?

**Review by lwcooper — APPROVED** [review]

File: test/project/openapi.yml (hunk 2 above)
Comment (wrumsby) [reply test/project/openapi.yml:146]: Yep

File: src/ServerlessOpenApiDocumentation.ts (hunk 1 above)
Comment (wrumsby) [reply src/ServerlessOpenApiDocumentation.ts:1]: This broke linting... somehow

Comment (lwcooper) [reply src/ServerlessOpenApiDocumentation.ts:1]: ah ok, fair enough. weird!

**Pull request merged**